### DISCUSSION TITLE
glTFForUE4 for Linux

### DISCUSSION THREAD

**Discussion by carlagx** [code4game/glTFForUE4#54] [General]
Are there efforts to compile the plugin for also for Linux?
The standard beta gltf import plugin, only crashes for me :(

**Comment by jixingcn** [comment]
The early `glTFForUE4` was evaluated in Ubuntu 20.04, but I don't have time to keep it working for Linux.

Do you mean `glTF Import` that is contained in the official version?
Sorry, I don't know and check it when I am free.

**Reply by carlagx** [reply]
thank you for the answer. I have a problem with a map in Carla. I can import it on Windows, but not on Linux (with the beta glTF importer). So I wanted to try your importer. I still clarify if it could be because the map was created with RoadRunner under Windows. If there is time and it does not make too much trouble, you can compile your importer for linux, otherwise I have to live without gltf and be satisfied with fbx :)
